docs/topics/proc_running-the-secured-booster-integration-tests-on-sb-vx.adoc

```diff
@@ -1,32 +1,57 @@
 
-[#running-the-secured-booster-integration-tests-on-vertx]
-= Running the {name-mission-secured} Booster Integration Tests on {VertX}
+[#running-the-secured-booster-integration-tests-on-{parameter-runtime}]
+= Running the {name-mission-secured} Booster Integration Tests on {runtime}
+//TODO: specify parameter in master file
 
 .Prerequisites
 
+
 * The `oc` client authenticated.
+// Should probably include the prerequisite to create a separate project for IT's in OpenShift
 
 .Procedure
 
+ifdef::secured-spring-boot-mission[]
+
+[IMPORTANT]
+--
+Keep the contents of the `service.sso.yaml` and the `.openshiftio/service.sso.yaml` templates synchronized to ensure that the integration tests work correctly.
```

Comment:
Do we actually expect end users to modify these files? This is just a note for booster authors I believe.

Reply:
@Ladicek You're right. I don't think users need to modify them in any way. If you see no reason for us to keep the not, I'm ok with removing it.